openshift/template.yaml

```diff
@@ -71,6 +124,16 @@ parameters:
 - name: IMAGE_PULL_POLICY
   value: "Always"
 - name: SUSPEND
-  value: "false"
+  value: "true"
```

Comment:
```suggestion
  value: "false"
```

Reply:
it could be worth to rename
`SUSPEND` into `PROW_JOBS_SCRAPER_SUSPEND`, `LOG_LEVEL` into `PROW_JOBS_SCRAPER_LOG_LEVEL` and `SUSPEND` into `PROW_JOBS_SCRAPER_SCHEDULE` so it makes it clear to which CronJobs these variables belong to.